srcpkgs/clion/template

```diff
@@ -0,0 +1,58 @@
+# Template file for 'clion'
+_dlname="CLion"
+_pkgname="clion"
+pkgname=clion
```

Comment:
Please use same case as tarball name. It is not only Void way, it also makes default update pattern work.